### ISSUE TITLE
`///` comments don't show up in Create/Find, etc args

### ISSUE THREAD

**Issue by ImLunaHey** [prisma/prisma#18644]
### Bug description

When adding a triple slash comment the comment is only added to the result but not the args for the model. This makes it harder to know what the fields are.

### How to reproduce

```prisma
model Category {
  id String @id @default(uuid())

  /// Whether this ticket category is enabled
  enabled Boolean @default(true)
}
```

```ts
await prisma.category.create({
    data: {
        enabled: true, // <-- Comment missing
    },
});

const x = await prisma.category.findFirst({
    where: {
        enabled: false, // <-- Comment missing
    }
})

if (!x) return;

x.enabled // <-- Has comment
```

### Expected behavior

All three instances of `enabled` to show the comment I added in the schema.

### Prisma information

See "steps to reproduce".

### Environment & setup

- OS:  macOS
- Database:  MySQL
- Node.js version:  v18.3.0

### Prisma Version

```
Environment variables loaded from .env
prisma                  : 4.11.0
@prisma/client          : 4.11.0
Current platform        : darwin
Query Engine (Node-API) : libquery-engine 8fde8fef4033376662cad983758335009d522acb (at node_modules/@prisma/engines/libquery_engine-darwin.dylib.node)
Migration Engine        : migration-engine-cli 8fde8fef4033376662cad983758335009d522acb (at node_modules/@prisma/engines/migration-engine-darwin)
Format Wasm             : @prisma/prisma-fmt-wasm 4.11.0-57.8fde8fef4033376662cad983758335009d522acb
Default Engines Hash    : 8fde8fef4033376662cad983758335009d522acb
Studio                  : 0.483.0
```

**Comment by jkomyno** [comment]
Hi @ImLunaHey, I know this is an old issue, I just wanted to tell you that this feature is already available in Prisma 5. Specifically, we recommend you to upgrade to the latest available version, Prisma 5.17.0. Thanks!

**Comment by ImLunaHey** [comment]
@jkomyno it would be nice if you closed the issue with the associated commit or PR.
